pedalboard/__init__.py

```diff
@@ -20,3 +20,6 @@
 
 for klass in AVAILABLE_PLUGIN_CLASSES:
     vars()[klass.__name__] = klass
+
+# pedalboard current version
+__version__ = '0.3.6'
```

Comment:
let's clean out this!

Reply:
I have. Have you seen my latest change?

Reply:
no need for `__version__` to be initialized here, you can do it in `version.py`